src/vg-tooltip.js

```diff
@@ -15,30 +20,44 @@
     // initialize tooltip with item data and options on mouse over
     vgView.on("mouseover.tooltipInit", function(event, item) {
       if (shouldShowTooltip(item)) {
-        init(event, item, options);
+        // clear existing promise because mouse can only point at one thing at a time
+        cancelPromise();
+
+        tooltipPromise = window.setTimeout(function() {
+          init(event, item, options);
+          tooltipActive = true;
```

Comment:
I didn’t put `tooltipActive` in `init()` or `clear()` because it’s only referenced by `onmouseover`, `onmousemove`, and `onmouseout` listeners, so putting it closer to the event listeners might be easier to read?